README.md

```diff
@@ -66,141 +66,25 @@ at https://www.azureiotsuite.com.
-of this demonstration, all the microservices are deployed in a single
-virtual machine; this is not considered a production-ready architecture.
-
-Our Standard deployment option should be used when you are ready to customize
-a production-ready architecture, built for scale and extensibility.
-
-Creating a Basic solution will result in the following Azure services being
-provisioned into your Azure subscription at cost: 
-
-| Count | Resource                       | Type         | Used For |
-|-------|--------------------------------|--------------|----------|
-| 1 | [Azure Active Directory Application][azure-active-directory] | | User Authentication |
-| 1 | [Linux Virtual Machine][virtual-machines] | Standard D1 V2 (1 core, 3.5GB memory) | Hosting microservices |
-| 1 | [Azure IoT Hub][iot-hub] | S1 – Basic tier | Device management and communication |
-| 1 | [Azure Cosmos DB][cosmos-db] | Standard | Storing configuration data, and device telemetry like rules, alarms, and messages |
-| 1 | [Azure Storage Account][storage-account] | Standard GRS | Storage for VM and Stream Analytics checkpoints |
-| 1 | [Web Application][web-application] | | Hosting front-end web application |
-
-## Standard
-The standard deployment is a production-ready deployment a developer can
-customize and extend to meet their needs. For reliability and scale, application
-microservices are built as Docker containers and deployed using an orchestrator
-([Kubernetes](https://kubernetes.io/) by default). The orchestrator is
-responsible for deployment, scaling, and management of the application.
-
-Creating a Standard solution will result in the following Azure services being
-provisioned into your Azure subscription at cost:
-
-| Count | Resource                       | Type         | Used For |
-|-------|--------------------------------|--------------|----------|
-| 1 | [Azure Active Directory Application][azure-active-directory] | | User Authentication |
-| 1 | [Azure Container Service][container-service] | | [Kubernetes](https://kubernetes.io/) orchestrator |
-| 4 | [Linux Virtual Machines][virtual-machines] | Standard D1 V2 (1 core, 3.5GB memory) | 1 master VM & 3 agents for hosting microservices with redundancy |
-| 1 | [Azure IoT Hub][iot-hub] | S1 – Basic tier | Device management and communication |
-| 1 | [Azure Cosmos DB][cosmos-db] | Standard | Storing configuration data, and device telemetry like rules, alarms, and messages |
-| 5 | [Azure Storage Accounts][storage-account] | Standard GRS | 4 for VM storage, and 1 for the Stream Analytics checkpoints |
-| 1 | [Web Application][web-application] | | Hosting front-end web application |
-
-> Pricing information for these services can be found
-[here](https://azure.microsoft.com/pricing). Usage amounts and billing details
-for your subscription can be found in the
-[Azure Portal](https://portal.azure.com/).
+## All done? Connect a Physical device!
```

Comment:
Does "All done?" still apply with the addition of the simulated device instructions?

Reply:
Just removed all done.